### PR TITLE
Deployment upgrade

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

**Comment by adamgraves-choices** [comment]
Still needs mongo environment variables to function correctly.

File: Dockerfile [hunk 1]
```diff
@@ -26,4 +26,4 @@ USER root
 RUN find /code -user 0 -print0 | xargs -0 chown $USERNAME:$USERNAME
 USER $USERNAME
 
-CMD [ "yarn", "run", "start" ]
+CMD [ "node", "server.js" ]
```
Comment (st3v3nhunt) [inline Dockerfile]: Can a comment be added here to explain why this is as it is so nobody tries to change it back to using Yarn to run something? Thanks!

**Comment by c2s-dev** [comment]
:rocket: deployment of nearby-services-api succeeded (http://nearby-services-api-pr-60.dev.beta.nhschoices.net)

File: Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ ENV NODE_ENV=${NODE_ENV}
 
 COPY yarn.lock package.json /code/
 
-RUN if [ "$NODE_ENV" == "production" ]; then yarn install --production --ignore-optional; else yarn install --ignore-optional ; fi
+RUN if [ "$NODE_ENV" == "production" ]; then yarn install --pure-lockfile --production --ignore-optional; else yarn install --pure-lockfile --ignore-optional ; fi
```
Comment (st3v3nhunt) [inline Dockerfile]: The `--pure-lockfile` option  doesn't need to be added

**Comment by c2s-dev** [comment]
:rocket: deployment of nearby-services-api succeeded (http://nearby-services-api-pr-60.dev.beta.nhschoices.net)

**Pull request merged**